### PR TITLE
TimescaleDB integration docs

### LINKED ISSUE #778: Document TimescaleDB integration for Sensu Enterprise
Reference: https://github.com/sensu/sensu-enterprise/pull/373

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

Adds documentation for the TimescaleDB integration in Sensu Enterprise 3.2

## Motivation and Context

Closes #778 

## Questions

- [x] Do we need to include any instructions around setting up TimescaleDB to work with Sensu?
- [x] Do we need any further context around "using a PostgreSQL connection"?
- [x] Should we create placeholder pages for other versions? (Ex: This integration is only available in version 3.2 and later, go here to upgrade...)

### CODE REVIEW

**Comment by cwjohnston** [comment]
We need to document the steps for creating and configuring a database to receive metrics. 

Following the TimescaleDB documentation, I started a TimescaleDB instance in a Docker container, then used `psql`  to run the following commands;

```
CREATE database sensu;

\c sensu

CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;

CREATE TABLE metrics (
    time TIMESTAMPTZ NOT NULL,
    name TEXT NOT NULL,
    value DOUBLE PRECISION NULL,
    source TEXT NOT NULL, 
    tags JSONB
);

SELECT create_hypertable('metrics', 'time');

CREATE USER sensu WITH PASSWORD 'secret';
GRANT ALL PRIVILEGES ON TABLE metrics TO sensu;   
```

This creates a database named `sensu` containing a table named `metrics`, creates a user named `sensu` and then grants that user all privileges on the `metrics` table. With this in place I configured Sensu Enterprise as described in this PR, added the `timescaledb` handler to an existing metric check and now can see data in TimescaleDB:

```
postgres=# \c sensu
You are now connected to database "sensu" as user "postgres".
sensu=# select * from metrics;
          time          |                   name                    |   value    |      source      | tags 
------------------------+-------------------------------------------+------------+------------------+------
 2018-10-10 03:19:22+00 | snmp.uap_ac_pro.interfaces.1.ifInOctets   |     196136 | sensu-enterprise | {}
 2018-10-10 03:19:22+00 | snmp.uap_ac_pro.interfaces.1.ifOutOctets  |     196136 | sensu-enterprise | {}
 2018-10-10 03:19:22+00 | snmp.uap_ac_pro.interfaces.2.ifInOctets   | 2565099687 | sensu-enterprise | {}
 2018-10-10 03:19:22+00 | snmp.uap_ac_pro.interfaces.2.ifOutOctets  |  600246939 | sensu-enterprise | {}
```

File: content/sensu-enterprise/3.2/integrations/timescaledb.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+- [Overview](#overview)
+- [Configuration](#configuration)
+  - [Example(s)](#examples)
+  - [Integration Specification](#integration-specification)
+    - [`timescaledb` attributes](#timescaledb-attributes)
+
+## Overview
+
+Send metrics to [TimescaleDB][2] using a PostgreSQL connection.
```
Comment (cwjohnston) [inline content/sensu-enterprise/3.2/integrations/timescaledb.md]: > Do we need any further context around "using a PostgreSQL connection"?

@apaskulin How about something like this?

"TimescaleDB is an open-source time-series database powered by PostgreSQL. This integration allows Sensu Enterprise to send metrics directly to TimescaleDB using the PostgreSQL protocol."

Comment (cwjohnston) [reply content/sensu-enterprise/3.2/integrations/timescaledb.md]: We can use https://www.postgresql.org/docs/current/static/protocol.html as a reference for PostgreSQL protocol if needed.

**Comment by apaskulin** [comment]
@cwjohnston I hesitate to include your example above since we want to avoid documenting TimescaleDB/PostgreSQL in our docs. I propose omitting specific how-to-create-a-table instructions in favor of the more general statements added in https://github.com/sensu/sensu-docs/pull/788/commits/56a476b29bd4d889487676cf5274ee113a1047b5.

**Comment by cwjohnston** [comment]
> Should we create placeholder pages for other versions? (Ex: This integration is only available in version 3.2 and later, go here to upgrade...)

I don't think we've done this in the past, but I can see how it could help with SEO and discoverability. Aside from the additional effort, I don't really see a downside. 👍

**Comment by cwjohnston** [comment]
> we want to avoid documenting TimescaleDB/PostgreSQL in our docs. 

@apaskulin I definitely agree on this point, but I think we need more detail on our end. Although the database name and table are configurable, I believe the table must be created as described -- column names and value types must match exactly -- in order for the integration to function.

I haven't yet found a reference for creating tables on the Timescale docs site, so I think we'd need to link to Postgres docs for that.

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**